spring-cloud-azure-starters/spring-cloud-azure-starter-storage/README.md

```diff
@@ -0,0 +1,66 @@
+# Spring Cloud Azure Storage 
+
+The project implements Spring Resource abstraction for Azure Storage which allows you to interact with Azure Storage instances using Spring programming model.
```

Comment:
Azure Storage account have fours types. Currently, we only support `blob`. This should be specified.

Reply:
Maybe a link to azure storage blob here.